web/src/components/Search.vue

```diff
@@ -74,17 +72,15 @@ export default {
       this.$db('Location')
         .select({
           view: 'All',
-          fields: ['prefecture', 'region'],
+          fields: ['prefecture'],
```

Comment:
🤔 I just noticed that the region column is in the DB, but there are no values. Probably best to add it later and fill out the DB.